src/cli/cli_dataset.cpp

```diff
@@ -98,7 +98,7 @@ otError Dataset::Print(otOperationalDataset &aDataset)
 
     if (aDataset.mComponents.mIsChannelMaskPresent)
     {
-        mInterpreter.mServer->OutputFormat("Channel Mask: %08x\r\n", aDataset.mChannelMask);
+        mInterpreter.mServer->OutputFormat("Channel Mask: 0x%08x\r\n", aDataset.mChannelMask);
```

Comment:
Although `Channel Mask` is implemented as an `uint32_t`, but it is defined as a bit mask with variable length. Do we have a need to make this change?

Reply:
Thread 1.2 Section 8.10.1.18.1 defines the Channel Mask Entry for IEEE 802.15.4-2006 2.4 GHz PHY as a 4 byte value. If a new Channel Page is defined in the future, we will need to more more changes than just the Channel Mask. Thoughts?

Reply:
okay, I am fine with it.